apps/android/boot.js

```diff
@@ -197,6 +197,28 @@
           lastSteps = steps;
         }, event.int*1000);
       },
+
+
+      "listRecs": function() { // GB({t:"listRecs", id:"20230616a"})
+        let recs = require("Storage").list(/^recorder\.log.*\.csv$/,{sf:true}).map(s => s.slice(12, 21));
+        recs = recs.slice(recs.indexOf(event.id)); // FIXME: Currently does not handle if the last fetched log was deleted on the watch, so that there won't be any entry corresponding to `event.id`
```

Comment:
I want to do this with `findIndex()` instead, to find the first log that is newer than the provided id. The checking function should check the date and appended letter.

Reply:
Yes, that'd be nicer I guess - not that the current solution is bad though